homeassistant/components/sensor/point.py

```diff
@@ -6,10 +6,10 @@
 """
 import logging
 
-from homeassistant.components.point import (
-    DOMAIN as PARENT_DOMAIN, MinutPointEntity)
+from homeassistant.components.point import MinutPointEntity
 from homeassistant.components.point.const import (
     DOMAIN as POINT_DOMAIN, POINT_DISCOVERY_NEW)
+from homeassistant.components.sensor import DOMAIN as PARENT_DOMAIN
```

Comment:
The sensor domain should be imported as `DOMAIN`.
```suggestion
from homeassistant.components.sensor import DOMAIN
```

Reply:
better now?